components/components.css

```diff
@@ -0,0 +1,856 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+:root {
+  --main-color: #df1b6c;
+  --black: #121212;
+  --black-secondary: #252525;
+  --white: #e5e5e5;
+  --white-secondary: #b8b8b8;
+  --main-font: "Montserrat", sans-serif;
+  --secondary-font: "Roboto", sans-serif;
+  --p-x3: 0 3rem 0 3rem;
+  --p-x2: 0 2rem 0 2rem;
+  --p-x1: 0 1rem 0 1rem;
+  --border: 1px solid red;
+  --max: 1440px;
```

Comment:
--max should signify what it is max for, like --max-screen-width

Reply:
I should be more careful with naming. Thanks for bringing it to notice.